osu.Game/Screens/Play/PlayerSettings/PlayerSettingsGroup.cs

```diff
@@ -15,12 +16,36 @@ public PlayerSettingsGroup(string title)
         {
         }
 
+        private ScheduledDelegate hoverExpandEvent;
+
         protected override bool OnHover(HoverEvent e)
         {
+            updateHoverExpansion();
             base.OnHover(e);
 
             // Importantly, return true to correctly take focus away from PlayerLoader.
             return true;
         }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            if (hoverExpandEvent != null)
+            {
+                hoverExpandEvent?.Cancel();
+                hoverExpandEvent = null;
+
+                Expanded.Value = false;
+            }
+
+            base.OnHoverLost(e);
+        }
+
+        private void updateHoverExpansion()
```

Comment:
Why is this in its own method, and why is it scheduling with 0ms?

Reply:
> Why is this in its own method

no particular reason, I just moved the code out when testing it to be more flexible - will move back into `OnHover()`


> why is it scheduling with 0ms

I didn't realised there's a `Scheduler.Add()` which should suit this usecase better, or are you saying I shouldn't be using the scheduler?

Reply:
Also while you're here I'd like to direct your attention to the edited description wrt the expand buttons

Reply:
Why is it scheduling in the first place? You only need to schedule if there's a reason to do so.

Reply:
I referred to the implementation in `ExpandingContainer` but failed to see that the scheduler is only needed because a delay is required there